lib/beaker/host/unix/exec.rb

```diff
@@ -26,7 +26,7 @@ def get_ip
     if self['platform'].include?('solaris') || self['platform'].include?('osx')
       execute("ifconfig -a inet| awk '/broadcast/ {print $2}' | cut -d/ -f1 | head -1").strip
     else
-      execute("ip a|awk '/global/{print$2}' | cut -d/ -f1 | head -1").strip
+      execute("ip a | awk '/global/{print$2}' | cut -d/ -f1 | #{self['hypervisor'] == 'vagrant' ? 'tail' : 'head'} -1").strip
```

Comment:
@nwolfe yeah, we don't typically see usage of those OSes in this hypervisor, so we don't test them ourselves. If there is a more widespread need for this change, I expect a ticket will be filed, then we can move this to a case that includes those two. For now, I think this is fine where it is.

@rishijavia, I'm not a fan of the line length using the ternary, but that's a small nit that won't block merging.